apps/st2-actions/template.html

```diff
@@ -184,7 +184,7 @@ <h2 class="st2-details__panel-title">
                   <div class="st2-label" status="record.status"></div>
                 </div> -->
                 <div class="st2-flex-table__column st2-actions__details-column-time">
-                  {{ record.start_timestamp | date:'medium':'UTC' }}
+                  {{ record.start_timestamp | date:'EEE, dd MMM y HH:mm:ss UTC':'UTC' }}
```

Comment:
I was thinking about it but decided to postpone until we find a way to switch between TZ